wsd/COOLWSD.cpp

```diff
@@ -4055,6 +4055,8 @@ void COOLWSD::cleanup([[maybe_unused]] int returnValue)
         FileRequestHandler.reset();
         JWTAuth::cleanup();
 
+        Util::forcedExit(returnValue);
```

Comment:
ah, and then there is some (currently unknown) glitch with the cleanup path so bail out early for now

Reply:
Exactly. I'm unable to reproduce the double-free reported (on jenkins) with asan. I had found and fixed one (accessing `getWakeupsArray()`, which returns a reference to a static in `SocketPoll::removeFromWakeupArray()` after said static had been destroyed during exit), which wasn't reproducible with 24.04 but easily was with 25.05 (as odd as that is).